libs/StatusGoQt/src/StatusGo/Messages/InputMessage.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include "MessageDto.h"
+
+#include <nlohmann/json.hpp>
+
+#include <QString>
+
+using json = nlohmann::json;
+
+namespace Status::StatusGo::Messages
+{
+
+struct InputMessage
+{
+    QString messageText;
+    QString chatId;
+    ContentType contentType = ContentType::Unknown;
+    QString replyTo; // Id of the message that we are replying to
+    QString ensName; // Ens name of the sender
+
+    static InputMessage createTextMessage(const QString &message, const QString &chatId);
+};
+
+void to_json(json& j, const InputMessage& d);
```

Comment:
Don't we use camelcase?

Reply:
@noeliaSD this is a requirement of [nohlman json](https://github.com/nlohmann/json) that we use for hassle-free, c++11 compatible json library. See [library example](https://github.com/nlohmann/json#basic-usage)